.github/workflows/integration_tests.yml

```
@@ -0,0 +1,154 @@
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  detect-changes:
+    runs-on: ubuntu-latest
+    outputs:
+      borg: ${{ steps.changes.outputs.borg }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)